tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/variabledef/TupleVariableDefinitionTest.java

```diff
@@ -38,12 +38,14 @@
  */
 public class TupleVariableDefinitionTest {
 
-    private CompileResult result, resultNegative;
+    private CompileResult result, resultNegative, tupleVarDefnSyntaxNegative;
 
     @BeforeClass
     public void setup() {
         result = BCompileUtil.compile("test-src/statements/variabledef/tuple-variable-definition.bal");
         resultNegative = BCompileUtil.compile("test-src/statements/variabledef/tuple-variable-definition-negative.bal");
+        tupleVarDefnSyntaxNegative = BCompileUtil
```

Comment:
Please reassign this to null at @AfterClass. We once had an OOM issue as we holding the references

Reply:
Will update